gdnative-derive/src/lib.rs

```diff
@@ -164,6 +201,49 @@ pub fn profiled(meta: TokenStream, input: TokenStream) -> TokenStream {
 /// - `no_editor`
 ///
 /// Hides the property from the editor. Does not prevent it from being sent over network or saved in storage.
+///
+/// ### `#[methods]`
+/// Adds the necessary information to a an `impl` block to register the properties to be and methods with Godot.
```

Comment:
"to be and methods with Godot"... not sure what you meant exactly?

Reply:
That's a brain fart.

"register properties and methods with Godot" was what I meant to write 😆